### ISSUE TITLE
Difficult/expensive-to-change domain impacts

### ISSUE THREAD

**Issue by matthewgilliland** [holacracyone/Holacracy-Constitution#450]
4.1.2 Get Permission Before Impacting Domains says:

> In service of your Role, you have the authority to impact and control your Role's Domains.
> 
> You may also impact any Domain held by a Circle containing your Role and not further delegated, or any Domain such a Circle itself may impact. _But if you believe your impact will be substantially difficult or expensive to undo, you need to get permission._

The italicized portion could be read in two ways:

The first, which is the most intuitive reading for me because of the wording and paragraph structure, is that it only applies to undelegated domains held by circles containing your role or that the circle can impact.

The second is that it applies to both undelegated domains _and_ your role's domains. I believe this was the intention of the sentence, but I don't think I would know that if I didn't have more context on that rule (and the policy we had at HolacracyOne prior to that rule being adopted in v5).

This seems like a very risky lack of clarity.

I would suggest something like this:

> In service of your Role, and if you do not believe your impact will be substantially difficult or expensive to undo, you have the authority to impact and control your Role's Domains. You may also impact any Domain held by a Circle containing your Role and not further delegated, or any Domain such a Circle itself may impact. If you believe your impact will be substantially difficult or expensive to undo, you need to get permission.

Is there something I'm missing there?

**Comment by brianjrobertson** [comment]
@matthewgilliland The intention was actually the first of your two interpretations, which also seems the most intuitive one to me as well.  Given that, I'm not sure there's anything to clarify here, unless we're seeing people interpret it in the second manner, which would seem odd to me...

**Comment by matthewgilliland** [comment]
@brianjrobertson Ah, so it was intended not to apply to any domain that has been delegated to your role? In your comments in a meeting earlier today, you indicated that you thought this provision would apply to a particular role, but under that reading the provision would not apply, as while the decision is difficult/expensive to undo, it is also within a domain delegated to that role.

This rule seems to provide a good deal of safety in general for such decisions for exactly such an example, no?

**Comment by brianjrobertson** [comment]
@matthewgilliland That's right.  In the case of that meeting, I wasn't aware that the domain had been delegated all the way down to a role (as opposed to just down to the circle in question but not further); or at least I wasn't thinking about that in the moment.

This rule would provide lots of safety if it were extended even to decisions that are fully within a domain owned by a role, but it also creates some problems, and I'm not sure that safety is needed, so I'd like to defer considering further until I see a reason why the safety is needed.

**Comment by SamirSaidani** [comment]
Just a comment on the belief pattern found in the current Constitution: you might belief that something is true and this thing might be objectively false. Which means that beliefs might cause permanent damage on the organization and as stated by the rule, you have the right to do so if you believe it. 

I think we should get rid of the belief pattern: I don't see the value of it in the context of the Constitution. You are responsible of your action, whatever your beliefs.

**Comment by brianjrobertson** [comment]
@SamirSaidani I doubt that would change anything in a positive direction; if the constitution didn't say you can use your best judgment to interpret rules and act on that interpretation, would what someone do instead?  They'd either still use their judgment to interpret what the rules mean, because that's just the natural thing to do for anyone acting from an empowered place, or they'd freak out about the "right" interpretation and defer their power to someone else or a boss, which is worse that the current situation.  At least with the empowered thing to do being very explicit we avoid the disempowered stance, and can put constraints on the empowered stance, which are in there - you can't just interpret something any way you like, especially where Individual Initiative is concerned - there's several key constraints on that.  But if you see a case that's either real or at least seems reasonably likely to occur where those constraints aren't sufficient or this rule causes harm, please share, I'd love to consider it.

**Comment by SamirSaidani** [comment]
@brianjrobertson I'm questioning this sentence in particular : "If you believe that ..., then do ...".

It's obvious that in any game, we use our judgment to interpret rules and act on it. It's implicit for any rule. But in that particular case, _we're reifying the way we interpret the rule, and introducing it inside a rule_. The rule says that "If you believe that rule X is true, then you can do Y". It means that the rule gives you an explicit authorization to act on belief, which also means that your action is always true even if X is objectively false. Which also means that you cannot being held responsible for a wrong interpretation, since your interpretation will always be true in that particular case. If there is a trial, and we want to judge your responsibility for killing the organization, then you won't be responsible at all, since the organization has an explicit rule which authorized you to do whatever you want based on your belief. You are never wrong in such a system. It takes the judge out of the equation, since he has no way to interpret the rule "the impact must not be substantially difficult or expensive to undo" against your interpretation.

You can read this sentence "If you believe your impact will be substantially difficult or expensive to undo, you need to get permission." in its dual way: _"If you believe your impact will not be substantially difficult or expensive to undo, you don't need to get permission."_. This is a problem: it gives you unlimited power on the organization based on your belief. I've already gave you a specific real-life case in #435: an individual initiative based on a belief made a permanent damage on the organization AND the organization takes the responsibility out of you since the rule authorized you to act on your belief AND the organization has no way to put a firewall-rule to mitigate your action.

I think that the whole belief design pattern for the Constitution is harmful, and might create unavoidable permanent damage on the organization.
